service.py

```diff
@@ -492,20 +492,25 @@ def download(link, episode=""):
             xbmc.sleep(500)
             if sys.version_info.major == 3:
                 log(__name__, "Checking '%s' for subtitle files to copy" % (local_tmp_file))
-                if sys.platform == "linux" or sys.platform == "linux2":
+                if sys.platform == "linux" or sys.platform == "linux2" and not 'ANDROID_ARGUMENT' in os.environ:
```

Comment:
Can't we check in here if "linux" is a substring instead of checking for linux and linux2? Also, since this code appears in at least two places then let's extract a method and use that instead!